compiler/src/main/kotlin/io/spine/protodata/Context.kt

```diff
@@ -29,51 +29,27 @@ package io.spine.protodata
 import com.google.protobuf.compiler.PluginProtos.CodeGeneratorRequest
 import io.spine.base.EventMessage
 import io.spine.core.UserId
-import io.spine.protodata.subscriber.Subscriber
 import io.spine.server.BoundedContext
 import io.spine.server.BoundedContextBuilder
 import io.spine.server.integration.ThirdPartyContext
-import io.spine.server.projection.ProjectionRepository
 
 /**
  * A factory for the `ProtoData` bounded context.
```

Comment:
I would say that “ProtoData” is a brand. It's a name of a library. We need something closer “to the ground” for a context name. 

Consider an analogy, Microsoft Directory is a tool for managing users, it may have a contexts called Users, Permissions, Organizations.